test/modules/chips/components/ShareDataChip.test.js

```diff
@@ -4,17 +4,16 @@ import { modalReducer } from '../../../../src/store/modal/modal.reducer';
 import { TestUtils } from '../../../test-utils';
 import shareSvg from '../../../../src/modules/chips/components/assistChips/assets/share.svg';
 import { sharedReducer } from '../../../../src/store/shared/shared.reducer';
+import { fileStorageReducer } from '../../../../src/store/fileStorage/fileStorage.reducer';
 import { ShareDialogContent } from '../../../../src/modules/share/components/dialog/ShareDialogContent';
 import { QueryParameters } from '../../../../src/domain/queryParameters';
+import { EventLike } from '../../../../src/utils/storeUtils';
 
 window.customElements.define(ShareDialogContent.tag, ShareDialogContent);
 window.customElements.define(ShareDataChip.tag, ShareDataChip);
 
 describe('ShareDataChip', () => {
-	const defaultSharedState = {
-		fileSaveResult: null,
-		coordinates: []
-	};
+	const defaultState = { adminId: null, fileId: null, latest: new EventLike({ success: false }) };
```

Comment:
not needed, use `initialState` instead
`import { modalReducer, initialState } from '../../../../src/store/modal/modal.reducer';`